As adding Boost as build dependency would bring too much difficulties,
this patch disables tests with Boost dependencies.

Josef Dvoracek | Institute of Physics | Czech Academy of Sciences | 2019-12-04

diff -Nru ./swig-4.0.1.orig/Examples/test-suite/common.mk ./swig-4.0.1/Examples/test-suite/common.mk
--- ./swig-4.0.1.orig/Examples/test-suite/common.mk	2019-12-04 18:14:49.591089000 +0100
+++ ./swig-4.0.1/Examples/test-suite/common.mk	2019-12-04 19:38:48.703807000 +0100
@@ -208,7 +208,6 @@
 	director_protected_overloaded \
 	director_redefined \
 	director_ref \
-	director_smartptr \
 	director_thread \
 	director_unroll \
 	director_using \
@@ -272,11 +271,6 @@
 	langobj \
 	li_attribute \
 	li_attribute_template \
-	li_boost_shared_ptr \
-	li_boost_shared_ptr_attribute \
-	li_boost_shared_ptr_bits \
-	li_boost_shared_ptr_director \
-	li_boost_shared_ptr_template \
 	li_carrays_cpp \
 	li_cdata_cpp \
 	li_cpointer_cpp \
@@ -299,7 +293,6 @@
 	multiple_inheritance_abstract \
 	multiple_inheritance_interfaces \
 	multiple_inheritance_nspace \
-	multiple_inheritance_shared_ptr \
 	name_cxx \
 	name_warnings \
 	namespace_chase \
